HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -3712,18 +3714,186 @@ def self.get_installation_quality_cooling_coeff(f_chg)
     return qgr_values, p_values, ff_chg_values
   end
 
-  def self.get_installation_quality_heating_coeff(f_chg)
+  def self.get_charge_fault_heating_coeff(f_chg)
     if f_chg <= 0
-      qgr_values = [-0.0338595, 0.0202827, -2.6226343]
-      p_values = [0.0615649, 0.0044554, -0.2598507]
+      qgr_values = [-0.0338595, 0.0, 0.0202827, -2.6226343] # Add a zero term to combine cooling and heating claculation
+      p_values = [0.0615649, 0.0, 0.0044554, -0.2598507] # Add a zero term to combine cooling and heating claculation
     else
-      qgr_values = [-0.0029514, 0.0007379, -0.0064112]
-      p_values = [-0.0594134, 0.0159205, 1.8872153]
+      qgr_values = [-0.0029514, 0.0, 0.0007379, -0.0064112] # Add a zero term to combine cooling and heating claculation
+      p_values = [-0.0594134, 0.0, 0.0159205, 1.8872153] # Add a zero term to combine cooling and heating claculation
     end
-    ff_chg_values = [8.33]
+    ff_chg_values = [0.0, 8.33] # Add a zero term to combine cooling and heating claculation
     return qgr_values, p_values, ff_chg_values
   end
 
+  def self.get_airflow_fault_cooling_coeff()
+    # Cutler curve coefficients for single speed
+    cool_cap_fflow_spec = [0.718664047, 0.41797409, -0.136638137]
+    cool_eir_fflow_spec = [1.143487507, -0.13943972, -0.004047787]
+    return cool_cap_fflow_spec, cool_eir_fflow_spec
+  end
```

Comment:
Can you call this method from `set_cool_curves_ashp` so that we don't have the same coefficients two different places?